filebeat/config/config_test.go

```diff
@@ -33,7 +33,7 @@ func TestReadConfig(t *testing.T) {
 	assert.Equal(t, "/var/log/s*.log", prospectors[0].Paths[1])
 	assert.Equal(t, "log", prospectors[0].Input)
 	assert.Equal(t, 3, len(prospectors[0].Harvester.Fields))
-	assert.Equal(t, "1", prospectors[0].Harvester.Fields["review"])
+	assert.Equal(t, int64(1), prospectors[0].Harvester.Fields["review"])
```

Comment:
Note: fields are parsed as is and types are preserved when encoding to json. This is a change from old 'fields' support in filebeat.